dist/nix/quick-lint-js.nix

```diff
@@ -15,15 +14,25 @@ mkDerivation {
   pname = "quick-lint-js";
   version = builtins.head (builtins.split "\n" (builtins.readFile ../../version));
 
-  src = ../../.;
+  src = with pkgs; lib.cleanSourceWith {
+    filter = lib.cleanSourceFilter;
+    src = lib.cleanSourceWith {
+      filter = name: type: !(lib.hasSuffix ".nix" name);
+      src = ../../.;
+    };
+  };
   unpackPhase = null;
+  buildInputs = [ pkgs.python310 ];
 
-  nativeBuildInputs = with pkgs; [ cmake ninja ];
+  nativeBuildInputs = with pkgs; [
+    clang_11 #does not work with clang_12 or clang_7
```

Comment:
Shouldn't this dependency on `clang_11` only exist on Darwin (macOS), not on Linux?

Reply:
nix will bring clang_11 in both case and will use it only during build

Reply:
That seems wasteful on Linux where GCC is the default. If we don't have a good reason to use Clang on Linux, we should avoid Clang on Linux.

In my experience, GCC generates faster executables than Clang does. We should prefer GCC if it works and is the default anyway.